shared/chat/conversation/input/index.native.js

```diff
@@ -13,18 +13,28 @@ import type {Props} from '.'
 // See if there's a better workaround later
 
 class ConversationInput extends Component<void, Props, void> {
+  _setEditing(props: Props) {
+    if (!props.editingMessage || props.editingMessage.type !== 'Text') {
+      return
+    }
+
+    this.props.setText(props.editingMessage.message.stringValue())
+    this.props.inputFocus()
+  }
+
   componentWillReceiveProps(nextProps: Props) {
     if (this.props.editingMessage !== nextProps.editingMessage) {
-      if (nextProps.editingMessage && nextProps.editingMessage.type === 'Text') {
```

Comment:
the problem is this is being remade so its not a props change, its new props coming into a constructor. that really shouldn't be happening but this should still work in this case